src/DevBetterWeb.Core/Entities/Member.cs

```diff
@@ -57,7 +57,19 @@ internal Member(string userId)
     public decimal? CityLatitude { get; set; }
     public decimal? CityLongitude { get; set; }
     public List<BillingActivity> BillingActivities { get; set; } = new List<BillingActivity>();
+    public List<MemberVideo> Videos { get; private set; } = new List<MemberVideo>();
 
+    public void AddVideo(MemberVideo video)
```

Comment:
They're not actually adding a video, but are adding some stats about videos they've watched, right?

Reply:
changed to AddVideoProgress